### DISCUSSION TITLE
Help securing multiple clients

### DISCUSSION THREAD

**Discussion by francescopozzoni** [keycloak/keycloak#14609] [Q&A]
Hi everyone,
I need to secure two clients with a single keycloack instance. 

The customer asked me if it was possible to have a single landing page after the login with redirect links to the two applications, is this possible in some easy way or I need to write a third client that serves as dispatch page?

Thank you in advance.

**Comment by darius-m** [comment]
The account console already contains a list of client applications in the `Applications` tab. This should mean that you can extend or set up the account console to include links to your applications.

However, since Keycloak is a SSO platform, most services should redirect the users to Keycloak for authentication and then be redirected back to the application with the user's data. This flow usually means that users will not see a Keycloak page after authentication, which could defeat the purpose of the list. The account console itself is an OIDC client of Keycloak, so even if you rely on it to add links to each client, you will essentially rely on a third client as the dispatch.

As a note, in SAML there is such a thing as "IdP initiated SSO", where the Identity provider (keycloak) can redirect users to a specific client endpoint (and have the user's identification assertions included in the redirect) to access the client after authentication immediately, but OIDC may lack such functionality, so users would have initiate the login process themselves.
